include/swift/AST/ASTBridging.h

```diff
@@ -71,6 +73,82 @@ struct BridgedIdentifierAndSourceLoc {
   BridgedSourceLoc NameLoc;
 };
 
+class BridgedDeclBaseName {
+  BridgedIdentifier Ident;
+
+public:
+#ifdef USED_IN_CPP_SOURCE
+  BridgedDeclBaseName(swift::DeclBaseName baseName) : Ident(baseName.Ident) {}
+
+  swift::DeclBaseName unbridged() const {
+    return swift::DeclBaseName(Ident.unbridged());
+  }
+#endif
+};
+
+SWIFT_NAME("BridgedDeclBaseName.createConstructor()")
+BridgedDeclBaseName BridgedDeclBaseName_createConstructor();
+
+SWIFT_NAME("BridgedDeclBaseName.createDestructor()")
+BridgedDeclBaseName BridgedDeclBaseName_createDestructor();
+
+SWIFT_NAME("BridgedDeclBaseName.createSubscript()")
+BridgedDeclBaseName BridgedDeclBaseName_createSubscript();
```

Comment:
Can’t these be static functions of `BridgedDeclBaseName`. IIRC I used static members in `NameMatcherBridging` and it worked.

Similar for the other `create` functions here.